samples/04_gis_analysts_data_scientists/translating_story_map_from_one_language_to_another.ipynb

```diff
@@ -0,0 +1,4030 @@
+{
```

Comment:
<p>can we hyperlink story map with a relevant link  in this line (A story map is a web map that is created.. )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Made the necessary changes</p>